app/src/main/java/swen_anigans/mathematicfanatic/QuizResultsActivity.java

```diff
@@ -71,14 +71,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Star[] starValues = getStars(correct,total,starGroup.getChildCount());
         setStars(starViews,starValues);
-        if(picked == null) {
-            setRewardListeners();
-        } else {
-            LinearLayout slideUp = (LinearLayout) findViewById(R.id.slideuppanel);
-            pickedTV = ((TextView) slideUp.getChildAt(0));
-            slideUp.removeAllViews();
-            formRewardSelectedLayout(slideUp,picked);
-        }
```

Comment:
I believe it was there to try and maintain state if the user selected something then slid the frame down. But I'm not entirely positive. I'm pretty positive it is working as intended now.